index.html

```diff
@@ -12220,14 +12228,55 @@ <h3>Examples</h3>
 
 <section id="security" class="appendix">
   <h3>Security Considerations</h3>
-  <p class="ednote">Consider requirements from <a href="https://w3ctag.github.io/security-questionnaire/">Self-Review Questionnaire: Security and Privacy</a>.</p>
-  <p>See, <a href="#iana-considerations" class="sectionRef"></a></p>
+  <p>See, <a href="#iana-security">Security Considerations</a> in <a href="#iana-considerations" class="sectionRef"></a>.</p>
+  <p>Future versions of this specification
+    may incorporate subresource integrity [[?SRI]] as a means of ensuring that cached and retrieved
+    content matches data retrieved from remote servers; see <a href="https://github.com/w3c/json-ld-syntax/issues/86">issue 86</a>.</p>
+</section>
```

Comment:
I think this last paragraph should be of class note; it is definitely a non-normative statement.